src/PrestaShopBundle/Form/Admin/Type/TranslatorAwareType.php

```diff
@@ -53,11 +53,13 @@ public function __construct(TranslatorInterface $translator, array $locales)
     /**
      * Get the translated chain from key.
      *
-     * @param $key the key to be translated
-     * @param $domain the domain to be selected
+     * @param $key - the key to be translated
```

Comment:
Why, we never do that? :thinking:

Reply:
Not sure, there is even better approach to add the type(in this case string) instead of '-' . Added because phpstorm complains about parameter of class 'the' missing - it does not make sense - so needed to change.